docs/docusaurus.config.ts

```diff
@@ -131,15 +131,15 @@ const config: Config = {
       {
         name: "image",
         property: "og:image",
-        content: `${url}/img/dagger-factory-share.jpg`,
+        content: `${url}/img/daggernaut-carpenter-robots-share.jpg`,
       },
       {
         name: "author",
         content: "Dagger",
       },
       {
         property: "twitter:image",
-        content: `${url}/img/dagger-factory-share.jpg`,
```

Comment:
Are we using this older image? If not, we should delete it as well.

Reply:
Probably not, but maybe worth keeping for a while for the cached links?